admin-dev/themes/default/js/bundle/admin_parameters/performancePageUI.js

```diff
@@ -58,10 +58,10 @@ var PerformancePageUI = {
     displayMemcacheServers: function() {
         var CACHE_ENABLED = '1';
         var cacheEnabledInput = document.getElementById('form_caching_use_cache');
-        var cacheSelected = document.querySelector('input[name="form[caching][caching_system]"]:checked').value;
+        var cacheSelected = document.querySelector('input[name="form[caching][caching_system]"]:checked');
         var memcacheServersListBlock = document.getElementById('servers-list');
         var newServerBtn = document.getElementById('new-server-btn');
-        var isMemcache = cacheSelected === "CacheMemcache" || cacheSelected === "CacheMemcached";
+        var isMemcache = cacheSelected && (cacheSelected.value === "CacheMemcache" || cacheSelected.value === "CacheMemcached");
```

Comment:
are you sure? I don't see the value of this refactoring

Reply:
still, if it's ok for you I merge :) Thanks!

Reply:
This is more a bug fix than only a refacto.

Because no option was checked on my form, `document.querySelector('input[name="form[caching][caching_system]"]:checked')` returned `null`. Trying to get the `value` from it threw an error on the JS console.

Reply:
Ok, would you mind to fix the same issue here too ? https://github.com/Quetzacoalt91/PrestaShop/blob/c24f5dea384b45b6fb6c86410f40db7efc120b87/admin-dev/themes/default/js/bundle/admin_parameters/performancePageUI.js#L28 (some duplication, so probably the same bug?) and then ping me and I merge :)

Good job!

Reply:
I just commited that change. Thanks